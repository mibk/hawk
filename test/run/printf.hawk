BEGIN {
	a = []
	b = 3
	c = "Syd"
	d = [0, []]
	e = 0 == 0

	for _, v in [a, b, c, d, e, f] {
		printf "%V\n", v
	}

	printf "%V:%V\n", true, false

	printf "%v\n", d
}
