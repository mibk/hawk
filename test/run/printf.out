array
number
string
array
bool
undefined
bool:bool
[0, []]
